src/videojs.ads.js

```diff
@@ -467,19 +472,17 @@ var
           },
           'content-playback': {
             enter: function() {
-              player.trigger({
-                type: 'contentplayback',
-                triggerevent: fsm.triggerevent
-              });
-              // Make sure that the cancelPlayTimeout is cleared
+              // make sure that any cancelPlayTimeout is cleared
               if (player.ads.cancelPlayTimeout) {
                 clearImmediate(player.ads.cancelPlayTimeout);
                 player.ads.cancelPlayTimeout = null;
-                // Trigger playback if play was canceled and the player is paused.
```

Comment:
@bclwhitaker informed me that it's possible to hit this state without receiving a play request from the user if `adscancelled` or `aderror` is fired from one of the early states like `content-set`